jCOS/createRestrictions/createRestrictions.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:11
+
+LABEL Khaos Research Group <khaos.uma.es>
+
+WORKDIR /usr/local/src/
+COPY . /usr/local/src/
+
+# RUN ls -la /usr/local/src/input
```

Comment:
Here there should be a step to build the jar, here is an example on another project.

https://github.com/jMetal/Evolver/blob/main/Dockerfile

Reply:
Jar file compilation in Dockerfile was created.